ocaml/xapi-cli-server/record_util.ml

```diff
@@ -184,6 +184,14 @@ let host_operation_to_string = function
   | `apply_updates ->
       "apply_updates"
 
+let update_guidance_to_string = function
```

Comment:
It seems that we have both regular and polymorphic variants to describe guidance. Is this required?

Reply:
The polymorphic one is generated from the datamodel enum. There is no way around that one.